demonstrations/tutorial_adaptive_circuits.py

```
@@ -61,10 +61,15 @@
 """
 
 import pennylane as qml
-from pennylane import qchem
-from pennylane import numpy as np
+import jax
+import numpy as np
 import time
 
+from pennylane import qchem
+from jax import numpy as jnp
+
+jax.config.update("jax_enable_x64", True)
+
 symbols = ["Li", "H"]
 geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 2.969280527]])

```

Consider fixing the `UserWarning: Contains tensors of types {'autograd', 'jax'};` in this tutorial.

---

Silencing this means I have to redefine H later. (Because the H created here needs to use pnp since we use the old optimizers). it is a bit strange but that's the only way to silence the errors